src/main/java/org/jitsi/protocol/xmpp/colibri/ColibriConference.java

```diff
@@ -104,19 +104,63 @@
      * @throws OperationFailedException if channel allocation failed due to
      *                                  network or bridge failure.
      */
+    default ColibriConferenceIQ createColibriChannels(
+        boolean                         useBundle,
```

Comment:
i notice some places we're using this spacing style and others we're not/it's being removed?

Reply:
I only see this spacing style in a few places in jicofo (and not in other java projects), so I remove it in favor of the common one. Will fix this one.